Editor/Resources/EditorWindow/Pages/HelpAndDocumentationPage.uxml

```diff
@@ -11,48 +11,48 @@
         <ui:VisualElement class="divider"/>
         <ui:VisualElement class="separator separator--horizontal">
             <ui:VisualElement class="card card--small">
-                <ui:Image name="HelpPageReportIssueIcon" class="card__icon"/>
+                <ui:Image name="HelpPageReportIssueIcon" class="card__icon--small"/>
```

Comment:
Am I missing something? Why are we making all the icons small? The Github icon looks good in the screenshot, but the video/forums icons look small

Reply:
Fixed!